### PR TITLE
Pulled prismic data for footer

### LINKED ISSUE #7: Pull Data: Footer

### PR DESCRIPTION
**Pull request by jzhzhang66**
- Pulled initial prismic data for Footer component
- Added a function call to retrieve footer data in api.js (getFooter)
- Renders on Home page currently as an example of a page displaying footer information 
Screenshot 2021-03-14 202033

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    return (
+        <>
+        <br /> <br /> <br />
+        <h1>Footer</h1>
+        {contactEmail}
+        {renderRichText(contactInformationDescription)}
+        {renderRichText(contactTitle)} 
+        {facebookLink} <br />
+        {instagramLink} <br />
+        {twitterLink} <br />
```
Comment (isabellberzsenyi) [inline components/Footer.js]: Stying comment: let's indent these lines

**Review by isabellberzsenyi — CHANGES_REQUESTED** [review]
Just a minor styling comment, but other than that looks good

**Review by josephmannis — APPROVED** [review]
✨

**Review by isabellberzsenyi — APPROVED** [review]
✨

File: pages/index.js [hunk 2]
```diff
@@ -16,16 +18,19 @@ export default function HomePage({ homePageData }) {
       {renderRichText(featuredStoriesTitle)}
       {renderRichText(newsletterSignup)}
       {renderRichText(newsletterDescription)}
+      <Footer footerData={footerData}/>
```
Comment (jean-zhang) [inline pages/index.js:24]: Should probably put it in the _app.js file so that it can be across all the pages

Comment (jean-zhang) [reply pages/index.js:24]: Nevermind! We discussed it here: https://github.com/Scout-NU/thealleyway-site/pull/20#discussion_r595574938

**Pull request merged**